.github/workflows/makefile-test.yml

```diff
@@ -49,12 +49,17 @@ jobs:
 
     runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
         submodules: recursive
+    - uses: actions/setup-python@v4
+    - run: python -m pip install --upgrade pip
+    - run: cd shared/PSyclone && pip install .
     - name: Install dependencies
       run: >
         sudo apt update && sudo apt install build-essential gfortran
         opencl-headers ocl-icd-opencl-dev pocl-opencl-icd
     - name: Build system
       run: source compiler_setup/gnu.sh && make
+    - name: Build with code-generation
+      run: source compiler_setup/gnu.sh && make all_gen
```

Comment:
I see that the all_gen target calls nemolite_cpu, this seems wrong as it does not use psyclone and it is already called from the default target above. Can you have a look?

Reply:
Removed.